### PR TITLE
Fixed the profile picture rendering on the Event dashboard.

### LINKED ISSUE #1267: Profile Picture is not rendering on the Event Dashboard page.
**Describe the bug**
Profile Picture is not rendering on the Event Dashboard page.

**To Reproduce**
Steps to reproduce the behaviour:

1. Create an event.
2. Now click on Show Event Dashboard.
3. Now, you can see that the profile picture is not visible in the left drawer.

**Expected behavior**
The Profile Picture should be visible on the events dashboard page like all other pages.

**Actual behavior**
Profile Picture is not rendering.

**Screenshots**
image

**Additional details**
I would like to work on this.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by AmitSharma512**
**What kind of change does this PR introduce?**

Now, the Profile picture is rendered on the Event Dashboard page.

**Issue Number:**

Fixes #1267 

**Snapshots/Videos:**

image

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: src/components/LeftDrawerEvent/LeftDrawerEvent.tsx [hunk 1]
```diff
@@ -160,14 +159,10 @@ const leftDrawerEvent = ({
...
-              {userImage && userImage ? (
-                <img src={userImage} alt={`Profile Picture`} />
-              ) : (
-                <img
-                  src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
-                  alt={`Dummy User Picture`}
-                />
-              )}
+              <img
+                src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
```
Comment (rishav-jha-mech) [inline src/components/LeftDrawerEvent/LeftDrawerEvent.tsx:163]: This does not solves the issue, what will happen when a user will have a profile picture uploaded and he gets to see this initial image from Dicebear ?

Comment (AmitSharma512) [reply src/components/LeftDrawerEvent/LeftDrawerEvent.tsx]: @rishav-jha-mech I have updated the code.

**Comment by rishav-jha-mech** [comment]
@AmitSharma512 upload a profile picture of user and attach a screenshot whether the uploaded user image is being shown or not in Left Drawer

**Comment by AmitSharma512** [comment]
@rishav-jha-mech 
uploaded photo :
image

**Comment by rishav-jha-mech** [comment]
LGTM

**Review by rishav-jha-mech — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**